modules/core/src/main/java/org/locationtech/jts/geom/GeometryFactory.java

```diff
@@ -377,13 +377,21 @@ public MultiPoint createMultiPoint(Point[] point) {
    *
    * @param coordinates an array (without null elements), or an empty array, or <code>null</code>
    * @return a MultiPoint object
+   * @deprecated Use {@link GeometryFactory#createMultiPointFromCoords} instead
```

Comment:
Actually, don't want to deprecate this.  It is useful for programmatic usage, to not need a separate code path just for empty coordinates.

Reply:
But `createMultiPointFromCoords` is exactly the same?

Reply:
Ah right...  cancel that, was thinking of something else.